main.cpp

```diff
@@ -63,9 +62,55 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
     
+    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+
+    bool today = 1;
+    ignoreUnused(today);
+    bool price = 1;
+    ignoreUnused(price);  
+    bool greeting = 7;
+    ignoreUnused(greeting);
+
+    int Rachel = 3;
+    ignoreUnused(Rachel);
+    int Score = 89;
+    ignoreUnused(Score);
+    int limitation = 1000;
+    ignoreUnused(limitation);
     
+    char mark = 99;
+    ignoreUnused(mark);
+    char initial = 88;
+    ignoreUnused(initial);
+    char level = 89;
+    ignoreUnused(level);
+
+    double result = 8.89384848;
+    ignoreUnused(result);
+    double declination = 4.345;
+    ignoreUnused(declination);
+    double data = 898.3636;
+    ignoreUnused(data);
+
+    float snack = 3.70;
+    ignoreUnused(snack);
+    float calories = 500.30;
+    ignoreUnused(calories);
+    float gpa = 4.70;
+    ignoreUnused(gpa);
```

Comment:
- [ ] Declare them all first, and then pass them all to a single call of `ignoreUnused()` at the end.